feat: allow custom icon in chatbot toggle (#211)
patternfly/chatbot: packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotToggle/ChatbotToggle.md

```js file="./ChatbotToggleBasic.tsx" isFullscreen

```

> ### Custom closed icon 
> 
> A custom icon can be passed to the toggle. It's recommended that an SVG which uses `fill="currentColor"` be used to allow for the image to be visible in both light and dark theme.

Reviewer: ```suggestion
### Custom toggle icon 

A custom icon can be passed to the toggle. To ensure the icon is visible in both light and dark themes, use an SVG image and set `fill="currentColor"`.
```
